### ISSUE TITLE
Can not verify session with key-file on lower resolutions or scaled desktop (button missing)

### ISSUE THREAD

**Issue by freaky-m0** [vector-im/element-web#15692]
![2020-11-10 11_13_27-Window](https://user-images.githubusercontent.com/4473764/98660799-fda38600-2345-11eb-8c5b-2c1f454f4725.png)

### Description

On displays with a lower resolution than 1080p or with desktop scaling (windows) enabled, the button for the verification with key-file is missing.

### Steps to reproduce

- Login on a desktop with low resolution or desktop scaling enablend (windows)
- try to verfiy with key-file

Normally there is a button to upload the key-file for verification / e2e-encryption

Logs being sent: no

### Version information

- **Platform**: web & desktop

For the web app:

- **Browser**: Chrome
- **OS**: Windows
- **URL**: Private Server with 1.7.12

For the desktop app:

- **OS**: Windows
- **Version**: 1.7.12

**Comment by t3chguy** [comment]
There is no missing button, you have to go to another session of yours and accept the prompt there.

**Comment by freaky-m0** [comment]
![grafik](https://user-images.githubusercontent.com/4473764/98662051-85d65b00-2347-11eb-8b70-71af6b930b31.png)

???

If I login on the same server with a other desktop there is the button... and the only difference is the screen resolution or scaling option. We tested several clients.

**Comment by t3chguy** [comment]
On the same account?
I applied scaling, smaller windows, everything I could throw at it and is never affected by scaling.
If it doesn't show up that means that it wasn't set up and you can only verify using another device.

![image](https://user-images.githubusercontent.com/2403652/98662527-d2b63380-233f-11eb-87f9-13abd8cf789e.png)

**Comment by freaky-m0** [comment]
Same account, yes. Maybe the language makes a difference too? Or our custom name (something like "ABCDEF Chat") In my first screenshot the text with Element Web / Desktop is different. It goes to the next line. 

ELEMENT >
WEB

ELEMENT >
DESKTOP

ELEMENT > 
iOS

ELEMENT >
ANDROID

**Comment by t3chguy** [comment]
![image](https://user-images.githubusercontent.com/2403652/98663902-98e62c80-2341-11eb-9225-cf2ad3312295.png)

Even in German with crazy scaling

**Comment by Tiuasimo** [comment]
Had the same problem with 2 employees.
First one had scaling at 150%. After changing it to 100% and restarting the element-desktop the green text appeared.
Second one had resolution at 1600x900 in terminalserver. After I took over the terminalserver-session and changed it to 1080p the green text also appeared. (I did not re-login or restarted the windows-session)
So it has nothing todo with some restrictions.

Unfortunately I also can't reproduce it on my machine. I can't explain why it happened on these two clients.
